data/mods/Magiclysm/vehicleparts/vehicle_parts.json

```diff
@@ -4,5 +4,37 @@
     "id": "orichalcum_frame_break",
     "subtype": "collection",
     "entries": [ { "item": "orichalcum_lump", "count": [ 4, 6 ] }, { "item": "orichalcum_sliver", "count": [ 4, 6 ] } ]
+  },
+  {
+    "abstract": "mana_engine",
+    "copy-from": "engine_combustion",
+    "type": "vehicle_part",
+    "fuel_type": "mana",
+    "m2c": 60,
+    "//": "30% energy efficiency",
+    "description": "A combustion engine.  Burns gasoline fuel from a tank in the vehicle."
```

Comment:
Even if this is not visible in game, it would be good to have a proper description here. Especially since you didn't put any descripton in the subsequent engines, so they will be described as regular combustion engines.